src/plot_api/accessibility.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+import c2mChart from "chart2music";
+
+export function enable (gd) {
+
+    const c2mData = {};
+    const labels = [];
+    
+    const fullData = gd._fullData;
+
+    for(var i = 0; i < fullData.length; i++) {
+        var trace = fullData[i] ?? {};
+        var {type, x = [], y = [], name = i, text = []} = trace;
+        if(type === 'scatter') {
+            var traceData = [];
+            if ('y' in trace) {
+                for(var p = 0; p < y.length; p++) {
+                    traceData.push(
+                        {
+                            x: x.length > 0 ? x[p] : p,
+                            y: y[p],
+                            label: text[p] ?? p
+                        })
+                }
+                c2mData[name] = traceData;
+                labels.push(name);
+            }
+        }
+        else {
+            // 'Accessibility not implemented for trace type: ' + trace.type
+            return;
```

Comment:
Perhaps we should use `continue` instead of `return` here.

Reply:
Yes I think so. There's some overview text c2m reads out when you first focus on the chart, right? Perhaps we can have any skipped traces briefly mentioned there?

Reply:
I can look into this. Need to look into where that closed captions text element goes in general, not sure about outputting our own messages into it, it's more than just the overview at the start, also reads off the data as you navigate it.

But this general point is a good one, ideally want to be able to support more than just scatter traces and want to support figures containing unsupported traces alongside supported ones. 

Then there's also the opportunity for a config option to say which traces should be in accessibility mode

Reply:
@julianna-langston that would be great! I think that's going to be really important for this project, so that plotly.js and Dash users who care about this can turn on chart2music in all of their charts, knowing that the result will not always be complete but will always be as useful as it can be and will tell them what they're missing. And then over time as both the plotly.js and the chart2music sides add more features, existing applications will automatically improve.

Reply:
Chart2Music v1.12 now has an `unsupported` chart type. [Examples and documentation](https://chart2music.com/docs/API/Developer#unsupported-groups-of-data)